tests/wpt/web-platform-tests/css/cssom/stylesheet-replacedata-dynamic.html

```diff
@@ -0,0 +1,11 @@
+<!doctype html>
+<title>(Test #1) Stylesheet replaceData Dynamic</title>
```

Comment:
Let's call this: `CSS Test: Dynamic changes to the stylesheet contents using replaceData are reflected`